Bento/Decorators/Stylable.swift

```diff
@@ -0,0 +1,42 @@
+extension Renderable {
+    public func styling(_ styleSheet: StyleSheet<View>) -> AnyRenderable {
+        return StyledComponent(base: self, styleSheet: styleSheet)
+            .asAnyRenderable()
+    }
+}
+
+struct StyledComponent<Base: Renderable>: Renderable {
+    private let styleSheet: StyleSheet<Base.View>
+    private let base: Base
+
+    init(
+        base: Base,
+        styleSheet: StyleSheet<Base.View>
+    ) {
+        self.styleSheet = styleSheet
+        self.base = base
+    }
+
+    func render(in view: Base.View) {
+        base.render(in: view)
+    }
+
+    func didMount(to view: Base.View, storage: ViewStorage) {
+        base.didMount(to: view, storage: storage)
+
+        var view = view
```

Comment:
So even if it's a ref type we still need `var` dance?

Reply:
`StyleSheet` uses `inout` to support also modifying value types. So yes.